src/Tribe/iCal.php

```diff
@@ -197,48 +206,62 @@ public function maybe_add_link() {
 	 * Executes the iCal generator when the appropriate query_var or $_GET is setup.
 	 */
 	public function do_ical_template() {
-		// hijack to iCal template
-		if ( get_query_var( 'ical' ) || isset( $_GET['ical'] ) ) {
-			/**
-			 * Action fired before the creation of the feed is started, helpful to set up methods and other filters used
-			 * on this class.
-			 *
-			 * @since 4.6.11
-			 */
-			do_action( 'tribe_events_ical_before' );
+		// hijack to iCal template if query string included.
+		if (
+			! get_query_var( 'ical' )
+			&& ! isset( $_GET['ical'] )
+			&& ! get_query_var( 'outlook-ical' )
+			&& ! isset( $_GET['outlook-ical'] )
+		) {
+			return;
+		}
 
-			if ( ! $wp_query = tribe_get_global_query_object() ) {
-				return;
-			}
+		if (
+			get_query_var( 'outlook-ical' )
+			|| isset( $_GET['outlook-ical'] )
+		) {
+			$this->type = 'outlook';
+		}
+
+		/**
+		 * Action fired before the creation of the feed is started, helpful to set up methods and other filters used
+		 * on this class.
+		 *
+		 * @since 4.6.11
+		 */
+		do_action( 'tribe_events_ical_before' );
```

Comment:
What variables can we pass to the hook here?

Reply:
This is an existing hook, I could add another, but I prefer not to change this one.